exonum-java-binding/core/rust/src/lib.rs

```diff
@@ -27,8 +27,8 @@ extern crate exonum_cli;
 #[macro_use]
 extern crate exonum_derive;
 extern crate exonum_proto;
-extern crate exonum_rust_runtime;
-extern crate exonum_supervisor;
+pub extern crate exonum_rust_runtime;
```

Comment:
Does it say "reexport this crate"? Is that correct that unless you re-export, these crates will be inaccessible from the dependent crates?

Reply:
Yes, exactly.